style.css

```diff
@@ -1,2 +1,8 @@
 @import url(./components/spacing.css);
 @import url(./components/grid.css);
+@import url(./components/typography.css);
+
+:root {
```

Comment:
Preferably move `:root` to variables.css or root.css

Reply:
Only component-related variables will be here that's why I added it in this file.